src/themes/masschallenge/elements/container.variables

```diff
@@ -0,0 +1,8 @@
+/*******************************
+            Container
+*******************************/
+
+/*Devices*/
+@padding: 16px;
+@tabletBreakpoint: 768px;
+@tabletBreakpoint1: 769px;
```

Comment:
I think it is confusing to use two breakpoints with a similar name that do almost the same thing. Would it not be easier to use `@tabletBreakpoint: 769;`? When implemented in the `container.overrides` file as follows: `@media screen and (max-width: @tabletBreakpoint)` on line 27 and `@media screen and (min-width: @tabletBreakpoint)` on line 47, the result will be a change in the layout between 768px and 769px, which is the desired outcome.

Reply:
Let me fix that.
Since had initially thought that it would fix the tablet issue.
This makes alot of sense.